prow/plugins/override/override_test.go

```diff
@@ -587,6 +600,93 @@ func TestHandle(t *testing.T) {
 				},
 			},
 		},
+		{
+			name:    "successfully override prow job name",
+			comment: "/override prow-job",
+			contexts: map[string]github.Status{
+				"ci/prow/prow-job": {
+					Context:     "ci/prow/prow-job",
+					Description: "failed",
+					State:       github.StatusFailure,
+				},
+			},
+			presubmits: map[string]config.Presubmit{
+				"prow-job": {
+					JobBase: config.JobBase{
+						Name: "prow-job",
+					},
+					Reporter: config.Reporter{
+						Context: "ci/prow/prow-job",
+					},
+				},
+			},
+			jobs: sets.NewString("ci/prow/prow-job"),
+			expected: map[string]github.Status{
+				"ci/prow/prow-job": {
+					Context:     "ci/prow/prow-job",
+					State:       github.StatusSuccess,
+					Description: description(adminUser),
+				},
+			},
+		},
+		{
+			name:    "override same prow job and context",
+			comment: "/override prow-job\n/override ci/prow/prow-job",
+			contexts: map[string]github.Status{
+				"ci/prow/prow-job": {
+					Context:     "ci/prow/prow-job",
+					Description: "failed",
+					State:       github.StatusFailure,
+				},
+			},
+			presubmits: map[string]config.Presubmit{
+				"prow-job": {
+					JobBase: config.JobBase{
+						Name: "prow-job",
+					},
+					Reporter: config.Reporter{
+						Context: "ci/prow/prow-job",
```

Comment:
This test would already have passed before your change. Lets do this with a presubmit whose context is not the same as the one in the override command

Reply:
The test would have failed before it wouldn't accept "prow-job" as a valid override.
It was created mostly to verify both prow-job and the context names can be used together. But I'll change it as you asked